package.json

```diff
@@ -189,7 +189,7 @@
     "tree-kill": "1.2.2",
     "tsec": "0.2.1",
     "turbo": "1.2.9",
-    "typescript": "4.6.3",
+    "typescript": "4.7.2",
```

Comment:
Even explicitly setting the version here https://github.com/vercel/next.js/runs/6654336325?check_suite_focus=true fails because `4.6.3` is still being installed. :thinking:

Reply:
It seems that GitHub Action Cache is causing the problem:

<img width="1241" alt="image" src="https://user-images.githubusercontent.com/40715044/170992546-16300809-d973-4f19-987f-0e1604a3a909.png">

Here the cache is restored even though you have already changed the `package.json` and `pnpm-lock.yaml`